************* Module random
/log/firmware/firmware/usr/lib/python3.8/random.py:76:0: C0413: Import "import _random" should be placed at the top of the module (wrong-import-position)
/log/firmware/firmware/usr/lib/python3.8/random.py:94:4: W0231: __init__ method from base class 'Random' is not called (super-init-not-called)
/log/firmware/firmware/usr/lib/python3.8/random.py:103:0: W0613: Unused argument 'kwargs' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/random.py:140:12: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:141:16: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:142:16: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:143:12: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:173:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:233:12: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:235:12: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:244:4: C0103: Argument name "a" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:244:4: C0103: Argument name "b" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:250:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:253:8: W0621: Redefining name 'getrandbits' from outer scope (line 808) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:255:8: C0103: Variable name "r" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:257:12: C0103: Variable name "r" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:260:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:260:48: W0622: Redefining built-in 'int' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/random.py:266:8: W0621: Redefining name 'random' from outer scope (line 788) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:276:8: C0103: Variable name "r" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:278:12: C0103: Variable name "r" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:293:4: C0103: Argument name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:293:25: W0621: Redefining name 'random' from outer scope (line 788) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:361:8: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:393:8: W0621: Redefining name 'random' from outer scope (line 788) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:394:8: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:398:16: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:407:8: C0103: Variable name "hi" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:415:4: C0103: Argument name "a" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:415:4: C0103: Argument name "b" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:430:8: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:432:12: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:436:12: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:437:12: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:443:4: C0103: Argument name "mu" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:456:8: W0621: Redefining name 'random' from outer scope (line 788) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:458:12: C0103: Variable name "u1" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:459:12: C0103: Variable name "u2" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:460:12: C0103: Variable name "z" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:461:12: C0103: Variable name "zz" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:468:4: C0103: Argument name "mu" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:499:4: C0103: Argument name "mu" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:519:8: W0621: Redefining name 'random' from outer scope (line 788) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:523:8: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:524:8: C0103: Variable name "r" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:527:12: C0103: Variable name "u1" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:528:12: C0103: Variable name "z" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:530:12: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:531:12: C0103: Variable name "u2" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:535:8: C0103: Variable name "q" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:536:8: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:537:8: C0103: Variable name "u3" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:567:8: W0621: Redefining name 'random' from outer scope (line 788) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:547:4: R0914: Too many local variables (16/15) (too-many-locals)
/log/firmware/firmware/usr/lib/python3.8/random.py:579:16: C0103: Variable name "u1" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:582:16: C0103: Variable name "u2" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:583:16: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:584:16: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:585:16: C0103: Variable name "z" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:586:16: C0103: Variable name "r" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:599:16: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:600:16: C0103: Variable name "b" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:601:16: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:603:20: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:605:20: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:606:16: C0103: Variable name "u1" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:547:4: R0912: Too many branches (13/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/random.py:616:4: C0103: Argument name "mu" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:644:8: W0621: Redefining name 'random' from outer scope (line 788) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:645:8: C0103: Variable name "z" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:650:12: C0103: Variable name "z" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:679:8: C0103: Variable name "y" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:680:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/random.py:691:8: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:704:8: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:726:8: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:729:0: W0613: Unused argument 'args' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/random.py:729:0: W0613: Unused argument 'kwds' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/random.py:740:0: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:741:4: C0415: Import outside toplevel (time) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/random.py:747:4: C0103: Variable name "t0" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:749:8: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:754:4: C0103: Variable name "t1" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:748:8: W0612: Unused variable 'i' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/random.py:762:0: C0103: Argument name "N" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/random.py:45:0: C0411: standard import "from itertools import accumulate as _accumulate, repeat as _repeat" should be placed before "from _collections_abc import Set as _Set, Sequence as _Sequence" (wrong-import-order)
/log/firmware/firmware/usr/lib/python3.8/random.py:46:0: C0411: standard import "from bisect import bisect as _bisect" should be placed before "from _collections_abc import Set as _Set, Sequence as _Sequence" (wrong-import-order)
/log/firmware/firmware/usr/lib/python3.8/random.py:47:0: C0411: standard import "import os as _os" should be placed before "from _collections_abc import Set as _Set, Sequence as _Sequence" (wrong-import-order)

-----------------------------------
Your code has been rated at 7.57/10

